### ISSUE TITLE
Change proportional to tabular figures

### ISSUE THREAD

**Issue by pantonis** [itfoundry/Poppins#23]
I am using Poppins in my web app and I have numbers that refresh in real-time.
Problem when I am using Poppins font is that number is keep getting resized based on the number digits. This is very irritating with no good UX.
I have attached a moving gif that shows the problem. On the left I have Tahoma and on the right figure I have Poppins. Check the size of the figure on the right which is being getting resized all the time.

![number-problem](https://user-images.githubusercontent.com/3295086/35150029-3a55a306-fd21-11e7-8248-fd2d9c38b681.gif)

From what I was told in this thread [1421](https://github.com/google/fonts/issues/1421) Poppins don't have tabular figures but instead proportional.

Can you please change it and make it tabular please?

Thanks

**Comment by jsmestad** [comment]
Would love to see this added to Poppins

**Comment by JapanYoshi** [comment]
I coded [a tool that previews any given installed font's reaction to the CSS property ``font-variant-numeric``](https://codepen.io/japanyoshi/details/LYYgrYZ). It is supposed to enable proportional or tabular numerals according to the necessary context.

* Calibri, Roboto and Open Sans seem to work as expected.
* Arial has an issue with forgetting to *not* kern between the digit 1 and symbols.
* Poppins completely ignores the CSS property.

I think it is because Poppins lacks the OpenType features necessary to take advantage of them. OpenType features would let most users see proportional numerals that go along with text, but let savvy designers use tabular numerals in contexts like these.

**Comment by duncangeere** [comment]
I would really like to see this improvement.

**Comment by kauelima** [comment]
I know this is an old and probably forgotten feature request, but here I am in 2023 still wanting poppins to have tabular figures. Since we already have proportional ones, I would not kill it, but only add the tabular ones. At the same time

**Comment by SoWhy** [comment]
I, too, would love to see this. I have switched our design to use Poppins but with the numbers, especially the 1s, being so small, it's really problematic for text that contains multiple numbers

**Comment by SCIFENEFICS** [comment]
Its 2024 and I am hoping this can still happen.

**Comment by outlawpractice** [comment]
Agreed, great font but I don't want to add special css to substitute a monospace font everywhere a number shows up.  Perhaps support `font-variant-numeric: tabular-nums`, so people who like it the way it is are happy, and the rest of us get what we need?

**Comment by iamkartiknayak** [comment]
```
.digit-container {
            display: inline-block;
            width: 0.6em;
            text-align: center;
        }
```

Using this class for each digit container would make sure the text always renders without it's variable movement

**Comment by adrian-ijenko** [comment]
Check out my fork at [Here](https://github.com/adrian-ijenko/Poppins) it contains tabular-nums
